### DISCUSSION TITLE
Add more interactive elements and customizations to the Menu Cart

### DISCUSSION THREAD

**Discussion by DanielCyran** [elementor/elementor#27891] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Slide cart in Elementor Pro needs a refresh and new functions ^-^ the current one looks old and uninviting

![image](https://github.com/elementor/elementor/assets/96839460/3816bfa0-790b-4aec-9247-508032763817)

### Describe the solution you'd like.

Elementor PRO users need to refresh their old WooCommerce widgets

### Describe alternatives you've considered.

I'm asking for a modern slide cart widget with coupons, a bar for free delivery, etc... please, no example from the screenshot :)

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by toineenzo** [comment]
Could probably be done with the off canvas widget. But nested elements support for the mini cart widget, would be nice, but also more styling/content options.
